velox/docs/monthly-updates/december-2021.rst

```diff
@@ -0,0 +1,40 @@
+********************
+December 2021 Update
+********************
+
+Documentation
+-------------
+
+* Add :doc:`Aggregation-related optimizations <../develop/aggregations>` guide.
+* Add :doc:`json_extract_scalar <../functions/json>` function.
+* Update :doc:`Mathematical Functions <../functions/math>`.
+* Update :doc:`Comparison Functions <../functions/comparison>`.
+* Update :doc:`How to add an aggregate function? <../develop/aggregate-functions>` guide with new registry.
+* Update :doc:`How to add a scalar function? <../develop/scalar-functions>` guide with complex types.
+
+
+Core Library
+------------
+
+* Fix a bug in HashJoin filter to enable TPC-H query 19.
+* Add support for partition key aliases to Hive Connector.
+* Add support for conversions between Velox string and Arrow string.
+* Add support for 'timestamp with timezone'.
```

Comment:
It is not clear to me what change this describes. Would you clarify?

Reply:
https://github.com/facebookincubator/velox/pull/761 I missed adding "in :func:`to_unixtime` function"